packages/util-use-case/lib/adapters/toResult.test.mjs

```diff
@@ -0,0 +1,17 @@
+import test from '@zorko-io/tool-test-harness'
+import sinon from 'sinon'
+
+import {toResult} from './toResult'
+
+const result = {}
+const req = {}
+const res = {send: sinon.stub()}
+const deps = {}
+
+const toResultReturns = toResult(result, req, res, deps)
```

Comment:
strange naming, why not `result = toResult(result, req, res, deps)`, also move that expression to `test(...)`

or 

```
const expected = ...
actual  = toResult(result, req, res, deps)

// compare  expected and actual here
```

Reply:
because we already have variable result. Actualy we can do it without addition variables
toResult(result, req, res) === result